google-cloud-tools-plugin/testSrc/com/google/gct/idea/appengine/cloud/AppEngineCloudConfigurableTest.java

```diff
@@ -155,7 +156,7 @@ private void initCloudConfigurable() {
   private File createTempFile() throws IOException {
     TemporaryFolder tempFolder = new TemporaryFolder();
     tempFolder.create();
-    File executable = new File(tempFolder.newFolder("bin"), "gcloud");
+    File executable = new File(tempFolder.newFolder("bin"), SystemInfo.isWindows ? "gcloud.cmd" : "gcloud");
```

Comment:
sounds like a good idea.

On Thu, Mar 31, 2016 at 10:05 AM, meltsufin <notifications@github.com>
wrote:

> In
> google-cloud-tools-plugin/testSrc/com/google/gct/idea/appengine/cloud/AppEngineCloudConfigurableTest.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/557#discussion_r58058478>
> :
>
> > @@ -155,7 +156,7 @@ private void initCloudConfigurable() {
> >    private File createTempFile() throws IOException {
> >      TemporaryFolder tempFolder = new TemporaryFolder();
> >      tempFolder.create();
> > -    File executable = new File(tempFolder.newFolder("bin"), "gcloud");
> > +    File executable = new File(tempFolder.newFolder("bin"), SystemInfo.isWindows ? "gcloud.cmd" : "gcloud");
>
> I don't see any other references. If you make
> CloudSdkUtil.getSystemCommand() public, you can avoid the duplication of
> this check.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/557/files/4ed192e3e009802ce94ff923ed0ad34ac78d289c#r58058478>
>